media-libs/jpeg/jpeg-9a.recipe

```diff
@@ -65,6 +65,9 @@ BUILD_PREREQUIRES="
 	cmd:make
 	"
 
+defineDebugInfoPackage jpeg \
+	$libDir/libjpeg.so.9.1.0
```

Comment:
Hi Jerome, I think my commits got tangled up with someone else's. I am thinking about removing this fork and create a new fork. Is there better way?